src/Component/Tags.elm

```diff
@@ -0,0 +1,322 @@
+    oldForm /= newForm
+
+
+
+-- View --
+
+
+tagView : List TagField -> Int -> TagField -> Html Msg
+tagView tags id_ tag =
+    let
+        validationString =
+            if tag.isNameValidated then
+                validationError tags tag
+                    |> Maybe.map (\s -> p [ class "mt-1 text-red-500 text-xs italic" ] [ text s ])
+
+            else
+                Nothing
+    in
+    tr []
+        [ td [ class "p-4" ]
+            [ if tag.isNew then
+                div [ class "h-0 mb-12" ]
+                    [ input
+                        [ type_ "text"
+                        , id <| String.fromInt id_ ++ "-name"
+                        , value tag.name
+                        , classList [ ( "border-red-500", Maybe.isJust validationString ) ]
+                        , onInput <| TagNameChanged id_
+                        , onBlur <| TagNameOnBlur id_
+                        , autocomplete True
+                        , class "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                        ]
+                        []
+                    , Maybe.withDefault (text "") validationString
+                    ]
+
+              else
+                span [ class "h-0 mb-12 ms-3" ] [ text tag.name ]
+            ]
+        , td [ class "p-4" ]
+            [ div [ class "h-0 mb-12" ]
+                [ input
+                    [ type_ "text"
+                    , id <| String.fromInt id_ ++ "-value"
+                    , value tag.value
+                    , onInput <| TagValueChanged id_
+                    , autocomplete True
+                    , class "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                    ]
+                    []
+                ]
+            ]
+        , td [ class "p-4" ]
+            [ div
+                [ class "cursor-pointer h-0 mb-6"
+                , onClick <| RemovedTag id_
+                ]
+                [ trashIcon ]
+            ]
+        ]
+
+
+addTagButtonRow : List (Html Msg)
+addTagButtonRow =
+    [ tr [ class "border-t" ]
+        [ td [ colspan 3, class "text-center" ]
+            [ button
+                [ class "mt-4 mb-4 bg-blue-500 text-white font-bold py-2 px-4 rounded"
+                , onClick AddNewTagButtonClicked
+                ]
+                [ text "Add a new tag" ]
+            ]
+        ]
+    ]
+
+
+introductionText : String
+introductionText =
+    """This table displays all the tags for this user group where you can view, edit and remove them.
+    Add a new tag by entering a name and an optional value for your tag. Names are required to be unique
+    across different tags and can not be longer than 32 characters."""
+
+
+view : Model -> Html Msg
+view model =
+    let
+        isButtonDisabled =
+            not (isFormChanged model) || not (isFormValid model)
```

Comment:
On page load, the button is initially enabled.
This is unlike the behavior in the other two tabs, so I assume it's a bug.
Could you show us how to fix it?

Reply:
You *can* make changes, but there is no *need*.
We are mostly interested in having a starting point for discussing stuff with you, and I don't think that asking you to fix every detail of a piece of code that will never be used would be very respectful of your time.